src/services/artist.ts

```diff
@@ -24,4 +25,46 @@ export default class ArtistService implements ArtistServiceInteface {
   getRecommendation(id: string): ArtistRecommendation[] {
     return this.artistRecommendationRepo.getArtistRecommendations(id);
   }
+
+  createNewRecommendation(questions: Questions): String {
+    const artistRecommendation = {
```

Comment:
the response is not supposed to be hardcoded in the function. Place it in in a separate json file the way it is has been done in the endpoints and operations I added